R/pkg/R/client.R

```diff
@@ -44,12 +44,14 @@ determineSparkSubmitBin <- function() {
 }
 
 generateSparkSubmitArgs <- function(args, sparkHome, jars, sparkSubmitOpts, packages) {
+  jars <- paste0(jars, collapse = ",")
   if (jars != "") {
-    jars <- paste("--jars", jars)
+    jars <- paste0("--jars ", jars)
```

Comment:
This seems pretty subtle -- Can you add a comment just above saying `Leave a space after --jars`